snooty/rstspec.toml

```diff
@@ -711,7 +711,7 @@ type = "text"
 
 [role.rfc]
 help = """Reference an IETF RFC."""
-type = "explicit_title"
+type = {link = "https://tools.ietf.org/html/%s"}
```

Comment:
Apparently IETF does have a redirect set up, but technically there should be an `rfc` prefix; i.e.
`/html/rfc%s` e.g. https://tools.ietf.org/html/rfc7919#appendix-A.2